src/uct/api/v2/uct_v2.h

```diff
@@ -271,7 +271,13 @@ typedef enum {
      * in order for @ref UCT_MD_MEM_DEREG_FLAG_INVALIDATE flag to function
      * the @ref UCT_MD_MKEY_PACK_FLAG_INVALIDATE flag must be set.
      */
-    UCT_MD_MKEY_PACK_FLAG_INVALIDATE = UCS_BIT(0)
+    UCT_MD_MKEY_PACK_FLAG_INVALIDATE = UCS_BIT(0),
+
+    /**
+     * The flag is used to indicate that the memory region could be accessed
+     * by another process using the same MD to perform UCT operations.
```

Comment:
another process or another OS image ?

Reply:
by an another process which could run on the same CPU (the same OS image) or am another CPU from SmartNIC

Reply:
"using the same MD" -> "using the same device"